x-pack/plugins/fleet/public/applications/integrations/components/header/deployment_details.component.tsx

```diff
@@ -69,8 +69,7 @@ export const DeploymentDetails = ({ cloudId, learnMoreUrl, managementUrl }: Prop
     >
       <div style={{ width: 450 }}>
         <Description>
-          Send data to Elastic from your applications by referencing your deployment and
-          Elasticsearch information.
+          Send data to Elastic from your applications by referencing your deployment.
```

Comment:
one thing, this text is not yet translated, could you wrap it in `i18n.translate` like in [line 42](https://github.com/elastic/kibana/blob/dcca1f3c0bc44502f12027b8982638e023feac4f/x-pack/plugins/fleet/public/applications/integrations/components/header/deployment_details.component.tsx#L42) and give it a unique key e.g. `xpack.fleet.integrations.deploymentDescription`?

Reply:
@juliaElastic Thank you for the review. I wrapped the view deployment description into the `i18n.translate`. 
Please check it.